src/appshell/view/dockwindow/internal/dockwindowactionscontroller.cpp

```diff
@@ -37,6 +37,7 @@ void DockWindowActionsController::init()
     dispatcher()->reg(this, "set-dock-open", this, &DockWindowActionsController::setDockOpen);
     dispatcher()->reg(this, "toggle-dock", this, &DockWindowActionsController::toggleOpened);
     dispatcher()->reg(this, "toggle-floating", this, &DockWindowActionsController::toggleFloating);
+    dispatcher()->reg(this, "restore-default-layout", this, &DockWindowActionsController::restoreDefaultLayout);
```

Comment:
It would be great if all these actions started with `dock-`. So it would be easier to see what these actions are and there would be less of conflict. For example, `restore-default-layout` can also be in the context of score.

Reply:
renamed